app/controllers/facility_accounts_controller.rb

```diff
@@ -171,22 +179,22 @@ def accounts_receivable
   # GET /facilities/:facility_id/accounts/:account_id/statements/:statement_id
   def show_statement
     @facility = current_facility
-    action='show_statement'
-
-    case params[:statement_id]
-      when 'list'
-        action += '_list'
-        @statements = Statement.where(:facility_id => current_facility.id, :account_id => @account).order('created_at DESC').all.paginate(:page => params[:page])
-      when 'recent'
-        @order_details = @account.order_details.for_facility(@facility).delete_if{|od| od.order.state != 'purchased'}
-        @order_details = @order_details.paginate(:page => params[:page])
-      else
-        @statement=Statement.find(params[:statement_id].to_i)
-        @order_details = @statement.order_details
+
+    if params[:statement_id] == "list"
+      action = "show_statement_list"
+      @statements =
+        current_facility
+        .statements
+        .where(account_id: @account.id)
+        .paginate(page: params[:page])
+    else
+      action = "show_statement"
+      @statement = Statement.find(params[:statement_id])
+      @order_details = @statement.order_details
```

Comment:
Can you add `.paginate(:page => params[:page])` onto this PR? I need it in the credit card branch, but I'd rather avoid the conflicts.